lib/parsers/stream-json-parser.ts

```
@@ -0,0 +1,229 @@
+  private readonly disableTimeout: boolean;
+
+  constructor(options: { disableTimeout?: boolean } = {}) {
+    super({ 
+      objectMode: true,
+      readableObjectMode: true,
+      writableObjectMode: false 
+    });
+    this.disableTimeout = options.disableTimeout || process.env.NODE_ENV === 'test';
+  }
+
+  _transform(chunk: Buffer, encoding: string, callback: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)